tests/commands/test_commands.py

```diff
@@ -341,15 +343,74 @@ def json(self):
         return {'result': {'latest_block_height': self.height}}
 
 
-@pytest.mark.skip
-@patch('bigchaindb.config_utils.autoconfigure')
-@patch('bigchaindb.backend.query.store_validator_update')
-@pytest.mark.tendermint
-def test_upsert_validator(mock_autoconfigure, mock_store_validator_update):
-    from bigchaindb.commands.bigchaindb import run_upsert_validator
+# @pytest.mark.execute
+# @patch('bigchaindb.lib.BigchainDB.get_validators')
+# @pytest.mark.abci
+def test_upsert_validator_new_with_tendermint(b, priv_validator_path, user_sk, monkeypatch):
```

Comment:
It would be good to explicitly skip this test with `@pytest.mark.skip`

Reply:
Done